### DISCUSSION TITLE
Import error (adress or city is required)

### DISCUSSION THREAD

**Discussion by bubfcg** [code4recovery/12-step-meeting-list#1489] [Q&A]
When I import a CSV it gives me an error (in Dutch), translated it would be something like "adress or city is required". But there is an adress and city in the file. I also get the error when I try to import the template.csv.

Screenshot of the error:
![screenshot](https://github.com/user-attachments/assets/54a7ec15-981d-4395-aad3-33b728b20aea)

And I have added the csv file I tried to import.
[meetings (1).csv](https://github.com/user-attachments/files/16663726/meetings.1.csv)

**Comment by joshreisner** [comment]
try renaming your `Address` column to `Formatted Address` - i believe that will fix it. 

the translation is supposed to say address _and_ city is required, but formatted address is a way to specify both in a single column

**Comment by bubfcg** [comment]
Did not fix it. By the way, the file I try to import was exported from the website first. Made some bulk changes and tried to upload again, did not changed any column names.

**Reply by joshreisner** [reply]
ok, you're right, sorry about that! 

i think the problem is that it's using semicolons as separators. i think this might have happened when you made the bulk changes. i'd try to re-export and make sure it uses commas, and it should be good (with either column heading)
